ember/app/components/base-barogram.js

```diff
@@ -18,7 +18,7 @@ export default Ember.Component.extend({
   contests: null,
   elevations: [],
 
-  flotStyle: safeComputed('height', height => Ember.String.htmlSafe(`width: 100%; height: ${height}px;`)),
+  flotStyle: conditional('height', htmlSafe(tag`width: 100%; height: ${'height'}px;`)),
```

Comment:
The false case will be `undefined`. You may want `or(xxx, raw(''))` depending on what `safeComputed` was doing.